samples/triangle/main.c

```diff
@@ -21,20 +21,20 @@ static float     m_viewport[4][4];
 
 typedef struct {
     float pos[3];
-    float color[3];
+    float color[4];
 } __attribute__((packed)) ColoredVertex;
 
 static const ColoredVertex verts[] = {
-    //  X     Y     Z       R     G     B
-    {{-1.0, -1.0,  1.0}, { 0.1,  0.1,  0.6}}, /* Background triangle 1 */
-    {{-1.0,  1.0,  1.0}, { 0.0,  0.0,  0.0}},
-    {{ 1.0,  1.0,  1.0}, { 0.0,  0.0,  0.0}},
-    {{-1.0, -1.0,  1.0}, { 0.1,  0.1,  0.6}}, /* Background triangle 2 */
-    {{ 1.0,  1.0,  1.0}, { 0.0,  0.0,  0.0}},
-    {{ 1.0, -1.0,  1.0}, { 0.1,  0.1,  0.6}},
-    {{-1.0, -1.0,  1.0}, { 1.0,  0.0,  0.0}}, /* Foreground triangle */
-    {{ 0.0,  1.0,  1.0}, { 0.0,  1.0,  0.0}},
-    {{ 1.0, -1.0,  1.0}, { 0.0,  0.0,  1.0}},
+    //  X     Y     Z       R     G     B    A
+    {{-1.0, -1.0,  1.0}, { 0.1,  0.1,  0.6, 1.0}}, /* Background triangle 1 */
+    {{-1.0,  1.0,  1.0}, { 0.0,  0.0,  0.0, 1.0}},
+    {{ 1.0,  1.0,  1.0}, { 0.0,  0.0,  0.0, 1.0}},
+    {{-1.0, -1.0,  1.0}, { 0.1,  0.1,  0.6, 1.0}}, /* Background triangle 2 */
+    {{ 1.0,  1.0,  1.0}, { 0.0,  0.0,  0.0, 1.0}},
+    {{ 1.0, -1.0,  1.0}, { 0.1,  0.1,  0.6, 1.0}},
+    {{-1.0, -1.0,  1.0}, { 1.0,  0.0,  0.0, 0.5}}, /* Foreground triangle */
+    {{ 0.0,  1.0,  1.0}, { 0.0,  1.0,  0.0, 0.1}},
+    {{ 1.0, -1.0,  1.0}, { 0.0,  0.0,  1.0, 0.5}},
```

Comment:
Some more blue would be helpful - most of the triangle is hidden if blue is used for alpha